aiohttp/connector.py

```diff
@@ -628,6 +628,9 @@ def _release(
         if should_close or protocol.should_close:
             transport = protocol.transport
             protocol.close()
+            # Ignore the following protocol exceptions
```

Comment:
This should only happen because of that bug, right? So, we can remove it once the bug is fixed.

```suggestion
            # TODO: Remove once fixed: https://bugs.python.org/issue39951
```

Reply:
I hope so! I didn't fully understand from that discussion (bpo-39951) whether it's a bug or a lack of `ssl` module functionality. Christian Heimes said:

> Typical HTTP clients like curl send a close_notify and then shut down the TCP connection. HTTP servers may not reply with close_notify or may not wait for the client to confirm the server-side close notify.

And then:
> Python's ssl module does not support one-way close yet. The unwrap() methods always performs a blocking two-way shutdown. It requires cooperation from both parties.

Seems it might be solved by implementing `one-way` close `unwrap` in `ssl` module and then adding corresponding functionality to `asyncio.sslproto.SSLProtocol`.